etc/telegraf.conf

```diff
@@ -430,8 +430,42 @@
 #   # listen = ":9126"
 
 
-# # Configuration for the Riemann server to send metrics to
+# # Configuration for Riemann server to send metrics to
 # [[outputs.riemann]]
+#   ## Address of the Riemann server
+#   address = "localhost:5555"
+#
+#   ## Transport protocol to use, either tcp or udp
+#   transport = "tcp"
```

Comment:
two benefits:

1. we can use url.Parse to validate the provided URL
2. it's more consistent with the configuration of other plugins